utopian-io/v2.utopian.io
# Add beneficiaries to an article

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_Create an article with beneficiaries_

* Create an article and add 2 beneficiaries with a weight of 10%
* Save
* ...

**Expected result:** You should be redirected to the edit page with all the data

_Update article page_

* Edit an article

**Expected result:** You shouldn't be able to edit the beneficiaries

_Errors validation_

* Set weights with negative values or above 100%

**Expected result:** You should see the validation errors
